### ISSUE TITLE
🔥 [iOS] The name of the given podspec `RNFBAdmob` doesn't match the expected one `RNFBAdMob`

### ISSUE THREAD

**Issue by danleveille** [invertase/react-native-firebase#2836]
## Issue

When manually installing react-native-firebase/admob, RNFBAdmob's name in PodSpec file does not match the same capitalization as the one in the documentation. When you run `pod install --repo-update`, it causes the following error:

>[!] The name of the given podspec `RNFBAdmob` doesn't match the expected one `RNFBAdMob`

`RNFBAdmob` - In the podspec
`RNFBAdMob` - (capital M) In the [documentation](https://invertase.io/oss/react-native-firebase/v6/admob/ios)

I would fix this issue myself, but I figured it may be a problematic change. Not sure if the documentation should just be updated instead?

---

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [x] I'm using Pods and my Podfile looks like:

```ruby
  pod 'RNFBAdMob', :path => '../node_modules/@react-native-firebase/admob'

```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### Have you converted to AndroidX?

- [ ] my application is an AndroidX application?
- [ ] I am using `android/gradle.settings` `jetifier=true` for Android compatibility?
- [ ] I am using the NPM package `jetifier` for react-native compatibility?

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

```
System:
    OS: macOS Mojave 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i7-7567U CPU @ 3.50GHz
    Memory: 2.16 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.12.0 - /usr/local/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.11.3 - /usr/local/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 13.1, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
  IDEs:
    Xcode: 11.1/11A1027 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.1 => 0.61.1 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-git-upgrade: 0.2.7
```

- **Platform that you're experiencing the issue on**:
  - [x] iOS
  - [ ] Android
  - [ ] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `6.0.3`
- **`Firebase` module(s) you're using that has the issue:**
  - `react-native-firebase/admob`
- **Are you using `TypeScript`?**
  - `Y/N` & `VERSION`
  
</p>
</details>

**Comment by danleveille** [comment]
This issue still requires the community's attention.

**Comment by Salakar** [comment]
I believe this was fixed in #2922 by @mikehardy - can you upgrade to v6.1.0 which has this fix included. Thanks.
